framework_tvb/tvb/core/entities/file/files_helper.py

```diff
@@ -265,7 +263,7 @@ def move_datatype(self, datatype, new_project_name, new_op_id):
         Move H5 storage into a new location
         """
         try:
-            full_path = datatype.get_storage_file_path()
+            full_path = datatype.storage_path
```

Comment:
storage_path is no longer an attribute on neither datatype nor index entities. This method receives as argument a datatype index. We could find its storage_path using h5.h5_file_for_index()